samples/snippets/requirements.txt

```diff
@@ -1,3 +0,0 @@
-pytest==6.2.4
```

Comment:
Is this file being deleted intentionally?

Reply:
Yup - moved it to `requirements-test.txt` ([see here](https://github.com/googleapis/python-aiplatform/pull/578/files#diff-bc8fe5bfdfb4b4373668f263f7ddeccc0a02466f3f1aa9ffc29bfad5c3f3cb74R1))

Reply:
Ah I was more curious about `google-cloud-storage` and `google-cloud-aiplatform`. The noxfile is looking for these file names specifically (in the same directory as the samples) so you might see issues with dependencies not getting installed.

https://github.com/googleapis/python-aiplatform/blob/b478075efb05553760514256fee9a63126a9916f/samples/snippets/noxfile.py#L152-L158

Reply:
The reason for removing those was because the following lines in the noxfile install the `google-cloud-aiplatform` library from source which also includes `google-cloud-storage` as [a required dependency](https://github.com/googleapis/python-aiplatform/blob/b478075efb05553760514256fee9a63126a9916f/setup.py#L70). We have `INSTALL_LIBRARY_FROM_SOURCE` set to `True` in all the sample Kokoro configs.
https://github.com/googleapis/python-aiplatform/blob/b478075efb05553760514256fee9a63126a9916f/samples/snippets/noxfile.py#L160-L161

Reply:
^ Yep including the sample dependencies in `requirements.txt` is useful for local runs and any external developers.

Reply:
Sounds good, updated!